agent/config/runtime_test.go

```diff
@@ -7379,6 +7382,7 @@ func TestSanitize(t *testing.T) {
 			"CirconusSubmissionInterval": "",
 			"CirconusSubmissionURL": "",
 			"Disable": false,
+			"DisableCompatOneNine": false,
```

Comment:
Would lean towards renaming this flag to something more metric... focused?
`DisableMetricsCompat` may be good

Reply:
This config is contained within the telemetry block, so you have to configure it as `telemetry { disable_compat_1.9 = true }` if you want to set it. I believe this does a good job of scoping it to mean "only disables metrics" even if it looks a bit sketch in isolation.